### ISSUE TITLE
ArchiveId=3 it's Card, not just RomFS

### ISSUE THREAD

**Issue by mailwl** [citra-emu/citra#2515]
When AC is connecting, it compares EULA from config, and from SMDH icon. To load SMDH, it opens archive id 3, then opens file with Path=[Binary: 0200000069636f6e00000000]
looks like 2 - section number, and in the middle - name ("icon").

```
[   3.155553] Service.FS <Debug> core\hle\service\fs\fs_user.cpp:Service::FS::OpenArchive:443: archive_id=0x00000003 archive_path=[Empty]
[   3.156306] Service.FS <Trace> core\hle\service\fs\archive.cpp:Service::FS::OpenArchive:266: Opening archive with id code 0x00000003
[   3.192393] Service.FS <Debug> core\hle\service\fs\fs_user.cpp:Service::FS::OpenFile:69: path=[Binary: 0200000069636f6e00000000], mode=1 attrs=0
[   3.231121] Service.FS <Trace> core\hle\service\fs\archive.cpp:Service::FS::File::HandleSyncRequest:105: Read Path: [Binary: 0200000069636f6e00000000]: offset=0x0 length=14016 address=0x121000
[   3.233528] Service.FS <Error> core\hle\service\fs\archive.cpp:Service::FS::File::HandleSyncRequest:110: Reading from out of bounds offset=0x0 length=0x000036C0 file_size=0x3000
[   3.234085] Service.FS <Trace> core\file_sys\ivfc_archive.cpp:FileSys::IVFCFile::Read:90: called offset=0x0, length=0x36C0
[   3.248890] Service.FS <Trace> core\hle\service\fs\archive.cpp:Service::FS::File::HandleSyncRequest:160: Close Path: [Binary: 0200000069636f6e00000000]
```

**Comment by wwylele** [comment]
This is [partially documented on 3dbrew](https://www.3dbrew.org/wiki/Filesystem_services#Binary_LowPath)

>The 0x3 archive is an interface for the 0x2345678E archive with the current process programID+mediatype. The file lowpath is 3-words. These words are written to 0x2345678E-archive file_lowpath+0, with the rest of that lowpath set to all-zero(lowpath is different from archive 0x2345678A). 

(Though I think that `file_lowpath+0` should be `file_lowpath+8`)

File low path for 0x2345678E

|Index word 	|Description|
|-------------|-------|
|0 	|0 for NCCH data, 1 for savedata. The latter is only valid for FSPXI. Value 2 is allowed via archive 0x3, it's unknown what this is.|
|1 	|TMD content index / NCSD partition index.|
|2 	|**Type: 0=romfs(0 for non-NCCH as well), 1=exefs ".code"(?), 2=exefs "icon"/"banner"/"logo", 3=unknown, 4=unknown, 5=unknown.**|
|3-4 	|**Filename for ExeFS.**|

**Comment by wwylele** [comment]
Fixed by #2561
